tracing-subscriber/src/fmt/time/chrono_crate.rs

```diff
@@ -0,0 +1,57 @@
+use crate::fmt::format::Writer;
+use crate::fmt::time::FormatTime;
+
+/// Formats [local time]s and [UTC time]s with [formatter] implementations
+/// that use the [`chrono` crate].
+///
+/// [local time]: https://docs.rs/chrono/0.4.26/chrono/offset/struct.Local.html
+/// [UTC time]: https://docs.rs/chrono/0.4.26/chrono/offset/struct.Utc.html
+/// [`chrono` crate]: https://docs.rs/chrono/0.4.26/chrono/
+/// [formatter]: https://docs.rs/time/0.3/time/formatting/trait.Formattable.html
```

Comment:
we're not making use of this trait—converting `time`'s syntax to `chrono`'s is _way_ beyond any reasonable scope. We should probably allow users to specify format strings as [we did in the past](https://docs.rs/tracing-subscriber/0.2.25/tracing_subscriber/fmt/time/struct.ChronoUtc.html#method.with_format) before we removed `Chrono`.
```suggestion
```